docs/changes.rst

```diff
@@ -40,6 +40,10 @@ Released: not yet
 
 * Increased minimum version of zhmcclient to 1.0.0.
 
+* Added a '--secure-boot' option to the 'lpar scsi-dump' and 'partition update'
+  commands. It had already been supported by the 'lpar scsi-load' command.
+  (issue #206)
+
```

Comment:
@andy-maier  Do we need to specify user needs to have reasonably current firmware on the HMC and SE to support for latest secure boot image from distributors (via bundle update and if that update support that image version). ?

Now they don't have any mechanism to export the keys (key store support not yet available with z15).

Reply:
That would be good. But I don't have the details about the bundle version, do you have that?

Reply:
@andy-maier I can check with HMC-SE team about this latest support but again every time we need to update this section with latest linux versions support and HMC-SE bundle levels. I am just thinking if we make generic statement as below, will that be OK ?

For supported Secure Boot Linux distributions please check the relevant documentation and also it requires to have latest HMC-SE bundle levels.

Reply:
I found this blog by Eberhard Pasch from the zLinux team about secure boot: https://linux.mainframe.blog/secure-boot/. Based on that I don't think that new firmware levels are needed for new versions of the Linux distros, so if a minimum firmware version is needed, I would expect that not to change over time. I'll confirm that with Eberhard.

Reply:
I talked with Eberhard, and the mechanism works as follows:
- The FW does not have any key or signature related to secure boot.
- The distros sign the kernel and store the public key needed for verifying the signature in the image.
- The zipl tool when preparing the boot volume extracts the public key from the image and stores it in the boot record for use by the FW
- The FW when starting the partition in secure boot mode, verifies the signature.

I think we only need to state that the boot volume needs to be prepared for secure boot, and to reference the blog article for details.

Reply:
Support for the secure-boot attribute requires HMC API version 3.2. Not sure about the SE bundle level, though.

Reply:
I have updated the descriptions of the `--secure-boot` options accordingly.

Reply:
Thanks for the clarification @andy-maier